db/migrate/20230403091245_create_buses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateBuses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :buses do |t|
+      t.string :bus_name, null: false
+      t.string :bus_type, null: false
+      t.string :bus_brand, null: false
+      t.integer :bus_capacity, default:0, null: false
+      t.timestamps
+    end
+
+    add_index :buses, :bus_name, unique: true
```

Comment:
can't we do this while defining the column at the first place? 🤔

Reply:
IDK why but defining the column at the first place doesn't adding to schema. But now i added it in the first place also found out that it is best practice to add_index for each migration which speed up the query time.